showcase.html

```diff
@@ -495,47 +475,50 @@ <h3>Subscribe us on YouTube</h3>
+                            <a href="mailto:asetalias1@gmail.com" target="_blank">Drop us a message</a>
                         </li>
                         <li>
-                            <i class="fa icon-grey fa-github text-grey"></i>
-                            <a href="https://github.com/asetalias" target="_blank">Github</a>
+                            <i class="fa icon-grey fa-github text-black"></i>
+                            <a href="https://github.com/asetalias" target="_blank">Github, we ❤️ open-source</a>
                         </li>
                         <li>
-                            <i class="fa icon-grey fa-youtube text-red"></i>
-                            <a href="https://www.youtube.com/channel/UCKkyqEMLFW3jz-q3nJIFL3g" target="_blank">YouTube</a>
+                            <i class="fa icon-grey fa-twitter text-blue"></i>
+                            <a href="https://twitter.com/asetalias" target="_blank">Twitter, for all the latest updates</a>
                         </li>
                         <li>
-                            <i class="fa icon-grey fa-facebook text-blue"></i>
-                            <a href="https://www.facebook.com/asetalias/" target="_blank">Facebook</a>
+                            <i class="fa icon-grey fa-whatsapp text-green"></i>
+                            <a href="https://chat.whatsapp.com/Li2gDrZd9xzJIJdNvSzLbR" target="_blank">Whatsapp, join our public channel</a>
+                        </li>
+                        <li>
+                            <i class="fa icon-grey fa-telegram text-blue"></i>
+                            <a href="https://t.me/asetalias" target="_blank">Telegram, interact with our alumni</a>
                         </li>
                         <li>
-                            <i class="fa icon-grey fa-google-plus-square text-yellow"></i>
-                            <a href="https://groups.google.com/forum/#!forum/amity-linux-group" target="_blank">Mailing List</a>
+                            <i class="fa icon-grey fa-medium text-brown"></i>
+                            <a href="https://asetalias.in/blog" target="_blank">ALiAS Writes, read about the ALiAS Community</a>
                         </li>
                         <li>
                             <i class="fa icon-grey fa-linkedin-square text-blue"></i>
-                            <a href="https://www.linkedin.com/company/asetalias/" target="_blank">LinkedIn</a>
+                            <a href="https://www.linkedin.com/company/asetalias/" target="_blank">LinkedIn, join the largest FOSS community in Amity University</a>
+                        </li>
+                        <li>
+                            <i class="fa icon-grey fa-facebook text-blue"></i>
+                            <a href="https://www.linkedin.com/company/asetalias/" target="_blank">Facebook, join a group of 500+ hackers contributing to FOSS</a>
```

Comment:
The link is incorrect, please check. Also, the one below it for Instagram.

Reply:
Supremely, my bad. Resolved.